lib/ssl/src/ssl_handshake.erl

```diff
@@ -349,12 +349,12 @@ certify(#certificate{asn1_certificates = ASN1Certs}, CertDbHandle, CertDbRef,
           crl_check := CrlCheck,
           log_level := Level,
           depth := Depth} = Opts, CRLDbHandle, Role, Host) ->
-    
+
     ServerName = server_name(ServerNameIndication, Host, Role),
-    [PeerCert | ChainCerts ] = ASN1Certs,       
```

Comment:
This is a general comment about fixing whitespaces. We usually put them into a separate commit, it makes it easier to review the code.

Reply:
> This is a general comment about fixing whitespaces. We usually put them into a separate commit, it makes it easier to review the code.

OK